modules/enabling-additional-enabled-capabilities.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_additional_enabled_capabilities_{context}"]
+= Enabling the cluster capabilities by setting additional enabled capabilities
+
+As a cluster administrator, you can enable the cluster capabilities by setting `additionalEnabledCapabilities`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
+
+.Procedure
+
+. View the additional enabled capabilities by running the following command:
++
+[source,terminal]
+----
+$ oc get -o json clusterversion version | jq -c '.spec.capabilities.additionalEnabledCapabilities'
+----
+
++
+.Example output
+[source,terminal]
+----
+["openshift-samples"]
+----
+
+. To set the `additionalEnabledCapabilities`, run the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion/version --type merge -p '{"spec":{"capabilities":{"additionalEnabledCapabilities":["openshift-samples", "marketplace"]}}}'
+----
+
+[IMPORTANT]
+====
+It is not possible to disable a capability which is already enabled in a cluster. The cluster version Operator (CVO) continues to reconcile the capability which is already enabled in the cluster.
+====
+
+
+If you try to disable a capability, then CVO shows the divergent spec:
+[source,terminal]
+----
+oc get -o json clusterversion version | jq -r '.status.conditions[] | select(.type == "ImplicitlyEnabledCapabilities") | .status + " " + .reason + ": " + .message'
```

Comment:
@darshan-nagaraj Same as above regarding jq link, please.

Reply:
With jsonpath
```
oc get clusterversion version -o jsonpath='{.status.conditions[]}{"\n"}' | select(.type == "ImplicitlyEnabledCapabilities") | .status + " " + .reason + ": " + .message'
```

Reply:
^ verified but failed. The one below works on my end.

`oc get clusterversion version -o jsonpath='{.status.conditions[?(@.type=="ImplicitlyEnabledCapabilities")]}{"\n"}'`

The result is like:
```
{"lastTransitionTime":"2022-07-22T03:14:35Z","message":"The following capabilities could not be disabled: openshift-samples","reason":"CapabilitiesImplicitlyEnabled","status":"True","type":"ImplicitlyEnabledCapabilities"}
```

Reply:
fixed.